src/nft/components/common/SortDropdown/SortDropdown.tsx

```diff
@@ -51,7 +53,7 @@ export const SortDropdown = ({
       borderBottomLeftRadius={isOpen ? '0' : undefined}
       borderBottomRightRadius={isOpen ? '0' : undefined}
       height="44"
-      style={{ width: inFilters ? 'full' : mini ? 'min' : maxWidth ? maxWidth : '300px' }}
+      style={{ width: isCollectionStatsLoading ? 220 : width }}
```

Comment:
nit: you could add the loading conditional to the `width` const. Also, do you need to add `px` to the end of `200`?